layouts/partials/blog/hero.html

```diff
@@ -2,10 +2,11 @@
 {{ $desc   := .Params.description }}
 {{ $date   := dateFormat "January 1, 2006" .Date }}
 {{ $isMain := eq .Kind "section" }}
-<section class="hero is-medium">
+<section class="hero is-small">
```

Comment:
Personally, I'm not a fan of this aesthetically. I feel like the hero on the main page should be spacious and inviting.

Reply:
It's just too big, it takes up way too much space,